.github/workflows/master_build_merge.yml

```diff
@@ -143,6 +145,9 @@ jobs:
         with:
           python-version: 3.9
 
+      - name: Setup Go
+        uses: actions/setup-go@v2
+
       - name: Set up JDK 17
```

Comment:
Do we need JDK 17 Given our base image provide that already

Reply:
This is in ubuntu-latest that's why need separate installation